src/Ledger/GovernanceActions/Properties.agda

```diff
@@ -39,12 +38,12 @@ instance
   ... | .NewCommittee new rem q | Enact-NewComm p
     rewrite dec-yes
       (¿ ∀[ term ∈ range new ] term
-           ≤ (s .pparams .proj₁ .PParams.ccMaxTermLength +ᵉ e) ¿) p .proj₂
+           ≤ s .pparams .proj₁ .PParams.ccMaxTermLength +ᵉ e ¿) p .proj₂
```

Comment:
We still have to use `_+ᵉ_` around tactics? 😿

Reply:
This is unrelated to tactics, it's just that we still don't have `Epoch` in our class structure.